integration_test/robots/profile_information_robot.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:fluffychat/pages/chat_details/chat_details_view.dart';
+import 'package:fluffychat/pages/profile_info/profile_info_body/profile_info_contact_rows.dart';
+import 'package:fluffychat/pages/profile_info/profile_info_body/profile_info_header.dart';
+import 'package:fluffychat/widgets/avatar/avatar.dart';
+import 'package:fluffychat/widgets/twake_components/twake_icon_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:patrol/patrol.dart';
+import '../base/core_robot.dart';
+
+class ProfileInformationRobot extends CoreRobot {
+  ProfileInformationRobot(super.$);
+
+  PatrolFinder getBackIcon() {
+    return $(AppBar).$(TwakeIconButton);
+  }
+
+  PatrolFinder getTitle() {
+    return $(AppBar).$(Text);
+  }
+
+  PatrolFinder getAvatar() {
+    return $(ProfileInfoHeader).$(Avatar);
+  }
+
+  PatrolFinder getDisplayName() {
+    final count = $(ProfileInfoHeader).$(Text).evaluate().length;
+    if(count>2){
+      return $(ProfileInfoHeader).$(Text).at(count-2);
```

Comment:
The `count - 2` hack relies on something inherently fragile: coupling (the fact of 2 distinct part of the code shares a secret on data organisation or each other inner working).

Given so, random changes done in software part A might impact in an impredictable way software part B

Can't we find a better way to locate the Display Name?